OsmAnd/src/net/osmand/plus/activities/MapActivityActions.java

```diff
@@ -318,6 +318,8 @@ public void contextMenuPoint(final double latitude, final double longitude, fina
 				R.drawable.ic_action_search_light).reg();
 		adapter.item(R.string.context_menu_item_share_location).icons(
 				R.drawable.ic_action_gshare_dark, R.drawable.ic_action_gshare_light).reg();
+		adapter.item(R.string.context_menu_item_view_location).icons(
```

Comment:
I think the "view" menu item is optional.  It seems handy but not necessary.  Right now, everything but the `geo:` menu item is replaced by a standard Share menu with a list of apps, from which the user chooses where the text gets sent to.  The `geo:` menu item just enables sending a location to a different Map app, or at least that is all I was able to get it to do.  So if you don't want it, I'll just remove it.